### ISSUE TITLE
WAF_Policy: allow use of policy/use refering an existing ASM Policy on the BIG-IP

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-appsvcs-extension#863]
### Environment
 * Application Services Version: 3.52.0
 * BIG-IP Version: 17.1.1.3

### Summary
It would be expected that in the WAF_Policy class, one can refer to an existing ASM policy.
For example, the LTM policy rule's action refers to the WAF_Policy object, which refers to an existing (already installed on the BIG-IP) on the BIG-IP (pointer).  
```
[...]
            "actions": [
              {
                "event": "request",
                "type": "waf",
                "enabled": true,
                "policy": {
                  "use": "myPolicyA"
                }
              }
            ]
          }
        ],
        "strategy": "best-match",
        "class": "Endpoint_Policy"
      },
      "myPolicyA": {
        "class": "WAF_Policy",
        "policy": {
          "use": "/Common/policy_a"
        }
      },
[...]
```
### Expected Behavior
Scenario mentioned above works. The WAF_Policy class object is a simple pointer to an existing ASM policy.

### Actual Behavior
Not working, we get the following error:
`{"code":422,"errors":["/Tenant/App/myPolicyA/policy: should NOT have additional properties"],"declarationFullId":"","message":"declaration is invalid","declarationId":"691121"}
`

**Comment by JuergenMang** [comment]
You can use following snippet to assign an existing waf policy. There is no requirement to define a policy object.

```json
"vs_waf": {
    "class": "Service_HTTPS",
    "policyWAF": {
        "bigip": "/Common/asm-policy-name"
    }
}
```

**Comment by amolari** [comment]
@JuergenMang Indeed, we're using today what you've mentioned. But we have a special request/use-case where we would like to switch easily the declaration from using existing (on the BIG-IP) ASM policies to URL-fetched policies.

**Comment by sunitharonan** [comment]
Thanks Alexandre, we have created [AUTOTOOL-4470](https://jira.f5net.com/browse/AUTOTOOL-4470) and added to our backlog.

**Comment by Nagaraju-F5** [comment]
Hi All,

As per the AS3 stands, We use "bigip" property to refer to the existing BIG-IP object instead of the "use" property. So We recommend the user to use "bigip" instead of "use" in the "WAF_Policy" class.

"myPolicyA": {
        "class": "WAF_Policy",
        "policy": {
          "bigip": "/Common/policy_a"     // use "bigip "Instead of "use". 
        }
 },
...

**Comment by mdditt2000** [comment]
Resolved in AS3 55
